en/deploy-br-federation-manager.md

```diff
@@ -0,0 +1,251 @@
+export DATA_PLANE_SYMBOL="a"
+
+cat << EOF > {k8s-name}-kubeconfig
+apiVersion: v1
+kind: Config
+current-context: ${DATA_PLANE_SYMBOL}
+contexts:
+- name: ${DATA_PLANE_SYMBOL}
+  context:
+    cluster: ${CURRENT_CLUSTER}
+    user: br-federation-member-${DATA_PLANE_SYMBOL}
+    namespace: kube-system
+clusters:
+- name: ${CURRENT_CLUSTER}
+  cluster:
+    certificate-authority-data: ${CLUSTER_CA}
+    server: ${CLUSTER_SERVER}
+users:
+- name: br-federation-member-${DATA_PLANE_SYMBOL}
+  user:
+    token: ${USER_TOKEN_VALUE}
+EOF
+```
+
+> **Note:**
+>
+> The environment variable `$DATA_PLANE_SYMBOL` indicates the name of the data plane cluster, and you should provide a brief and unique name.
+> We use it as the context name of kubeconfig in the script above and the context name will be used as `k8sClusterName`
+> in the `VolumeBackup` and `VolumeRestore` CR.
+
+#### Merge multiple kubeconfig files to one
+
+If you follow the step above to generate kubeconfig, you may have multiple kubeconfig files. We should merge them to one kubeconfig file.
+
+Suppose that you have 3 kubeconfig files with file paths: `kubeconfig-path1`, `kubeconfig-path2`, `kubeconfig-path3`,
+and you want to merge them to one kubeconfig file with file path `data-planes-kubeconfig`. You can execute the command below to merge kubeconfig files.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+KUBECONFIG=${kubeconfig-path1}:${kubeconfig-path2}:${kubeconfig-path3} kubectl config view --flatten > ${data-planes-kubeconfig}
+```
+
+### Deploy BR Federation Manager in Control Plane
+
+You should select one Kubernetes cluster as control plane to deploy BR Federation Manager. You need only execute steps below in the control plane.
+
+#### Create CRD
+
+BR Federation Manager uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) to extend Kubernetes.
+Therefore, to use BR Federation Manager, you must first create the CRD, which is a one-time job in your Kubernetes cluster.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/federation-crd.yaml
+```
+
+#### Prepare Kubeconfig Secret
+
+You already have a kubeconfig file of data planes. Now, you need encode the kubeconfig file to a secret.
+Firstly, encode the kubeconfig file by `base64 -i ${kubeconfig-path}`. Secondly, put the output of first step to a secret object.
+**Note, the name of the secret and the data key of kubeconfig field must be equal to which is in the example below**.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: br-federation-kubeconfig
+type: Opaque
+data:
+  kubeconfig: ${encoded-kubeconfig}
+```
+
+#### Install BR Federation Manager
+
+This section describes how to install BR Federation Manager using [Helm 3](https://helm.sh/docs/intro/install/).
+
```

Comment:
Can you please explain the difference between **quick deployment** and **custom deployment**?